lib/check-node.js

```diff
@@ -3,10 +3,10 @@ const semver = require('semver')
 
 module.exports = (cb) => {
   exec('node --version', (error, stdout, stderr) => {
-    if (error) return cb(error)
+    if (error) return cb(error, null)
     const output = (stdout + stderr).replace(/(\r\n|\n|\r| )/gm, '')
     const sem = semver.valid(output)
     if (sem) return cb(null, sem)
-    cb(null, new Error(output))
+    cb(new Error(output), null)
```

Comment:
The changes here are unrelated to the rest, right? I’d suggest separating that out into a different commit/different PR, then.